### ISSUE TITLE
[request] getting things gnome

### ISSUE THREAD

**Issue by svdm-artix** [chaotic-aur/packages#735]
https://aur.archlinux.org/packages/gtg/

:D

**Comment by tio-trom** [comment]
Fails to build 

```
error: target not found: python-caldav
error: target not found: python-liblarch>=3.2
```

https://builds.garudalinux.org/repos/chaotic-aur/logs/filtered/dep-not-in-repo/gtg.log

**Comment by Technetium1** [comment]
Dependency is failing on a new version: [python-caldav.log](https://builds.garudalinux.org/repos/chaotic-aur/logs/python-caldav.log)

**Comment by tobixen** [comment]
Test broke probably due to new version of the icalendar library (it was passing when v0.10 was released).  I've looked into the test and concluded that the assert in the test code is wrong (the `VALUE=DATE-TIME` is optional and was inserted with the previous version of icalendar, but missing in the latest version).

I will have this fixed.  Version v0.10.1, with some other fixes as well, will hopefully be released towards the end of the week.
